r/tests/testthat/test-parquet.R

```diff
@@ -97,3 +97,11 @@ test_that("make_valid_version()", {
   expect_equal(make_valid_version(1.0), ParquetVersionType$PARQUET_1_0)
   expect_equal(make_valid_version(2.0), ParquetVersionType$PARQUET_2_0)
 })
+
+test_that("write_parquet() defaults to snappy compression", {
+  tmp1 <- tempfile()
```

Comment:
This test should start with `skip_if_not_available("snappy")` like https://github.com/apache/arrow/blob/master/r/tests/testthat/test-compressed.R#L23